activestorage/lib/active_storage.rb

```diff
@@ -68,6 +68,11 @@ module ActiveStorage
   mattr_accessor :replace_on_assign_to_many, default: false
   mattr_accessor :track_variants, default: false
 
+  def self.disk_service_enabled?
+    local_service = Blob.services.fetch(:local)
```

Comment:
That's a great point, thanks for pointing that out @lfalcao We'll double check that 👍 

Regarding the files not being loaded, we tried a few things with no success. We'll try a bit more, otherwise we'll move everything back to the same file.

Thank you for the tips, that's super helpful!